plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -273,6 +273,18 @@ public static void init() throws Exception {
         get().configure();
     }
 
+    /**
+     *  Reinit to load CASC config to temporary avoid issues
+     *  https://github.com/jenkinsci/configuration-as-code-plugin/issues/1026
+     *
+     * @throws Exception
+     */
+    @Initializer(after = InitMilestone.JOB_LOADED)
```

Comment:
😱 Not sure this is a good idea.

Reply:
I don't think this is a good idea, but it can be used as a solution to some of the problems now. The real approach should make the loading of casc stable and reliable.
I submitted this PR because the loading of casc should be idempotent, so this method should not introduce new problems now (maybe the loading time becomes longer?)